openfisca_france/model/prestations/minima_sociaux/ass.py

```diff
@@ -167,6 +167,36 @@ def revenus_tns():
 
         return result
 
+class ass_eligibilite_cumul_individu(Variable):
+    value_type = bool
+    label = u"Eligibilité au cumul de l'ASS avec un revenu d'activité"
+    entity = Individu
+    definition_period = MONTH
+    reference = u"https://www.legifrance.gouv.fr/eli/decret/2017/5/5/ETSD1708117D/jo/article_2"
+
+    def formula_2017_09_01(individu, period):
+        # liste des decalage de périodes à contrôler plus la période en cours
+        liste_decalage = range(-12, 0 + 1)
```

Comment:
`douze_mois_precents = [period.offset(offset) for offset in range(-12, 0 + 1)]` me parait plus clair et évite de multiples `period.offset(decalage)`.

Reply:
C'est rectifié